src/main/java/edu/harvard/iq/dataverse/authorization/providers/oauth2/OAuth2FirstLoginPage.java

```diff
@@ -239,11 +235,20 @@ public void validateUserName(FacesContext context, UIComponent toValidate, Objec
         String userName = (String) value;
         logger.log(Level.FINE, "Validating username: {0}", userName);
         boolean userNameFound = authenticationSvc.identifierExists(userName);
+        boolean userNameValid = !userName.isBlank() && UserNameValidator.isUserNameValid(userName, null);
```

Comment:
Can `!userName.isBlank()` ever be `false`? I would assume not, as `UserNameValidator.isUserNameValid(userName, null)` checks that it contains between 2 and 60 chars.